src/audio/pipeline/pipeline-graph.c

```diff
@@ -532,7 +530,9 @@ struct comp_dev *pipeline_get_dai_comp_latency(uint32_t pipeline_id, uint32_t *l
 		if (ret < 0)
 			return NULL;
 
-		*latency += input_data / input_base_cfg.ibs - output_data / output_base_cfg.obs;
+		if (input_data && output_data)
```

Comment:
if  comp_get_attribute is not support by source or sink, the input_base_cfg & output_base_cfg are not initialized. There will be zero-division exception when input_base_cfg & output_base_cfg  are zero filled

Reply:
if the `comp_get_attribute` is not supported then the base_cfg will be not touched, it will remain un-initialized, likely 0, but by definition it is not safe to touch it.
Which means that if the `comp_get_attribute()` returns 0, the value which is passed uninitialized is not safe to touch, 0 means **success** or **untouched**.

Therefore, `input_base_cfg` and `output_base_cfg` must be initialized to either 0 and have a check for ibs and obs.

Reply:
agree, good catch. This is a bug that already exists on "main" independent of this PR. A fix added to the PR